roles/initrd/provision_initrd/files/local

```diff
@@ -100,39 +100,40 @@ mountroot()
     # Is this necessary?
     modprobe tmpfs
     modprobe squashfs
-    modprobe overlayfs
-    mkdir upper lower work # upper=overlay, lower=base squashfs, work=empty for new command syntax, mount on ${rootmnt}
-    mount -t tmpfs tmpfs upper work # put overlay files into a tmpfs so they don't get blown away on switch_root
-	CB=$(cat /proc/cmdline)
-	CB="${CB##*API_CB=}"
-	CB="${CB%% *}"
-	BASEFS=$(cat /proc/cmdline)
-	BASEFS="${BASEFS##*BASEFS=}"
-	BASEFS="${BASEFS%% *}"
-	OVERLAYFS=$(cat /proc/cmdline)
-	OVERLAYFS="${OVERLAYFS##*OVERLAYFS=}"
-	OVERLAYFS="${OVERLAYFS%% *}"
-	echo "downloading squashfs from http://${CB}/${BASEFS}"
-	logger "downloading squashfs from http://${CB}/${BASEFS}"
-	curl http://${CB}/${BASEFS} -o base.squashfs.img
+    modprobe overlay
+    mkdir outer
+    mount -t tmpfs tmpfs outer # put overlay files into a tmpfs so they don't get blown away on switch_root
+    mkdir lower outer/upper outer/work # upper=overlay, lower=base squashfs, work=empty for new command syntax, mount on ${rootmnt}
```

Comment:
Some extra diff below since I normalized the indentation, but the changes that matter are here (103-106) and below from lines 121-129.